src/data.json

```
@@ -8,16 +8,16 @@
     "from": "Nov, 2021",
     "to": "Current",
     "desc": [
-      "Developed multiple features for internal \"Auto Recon Tool\", handling reconciliation of millions of txns for payments, refunds, chargebacks and settlements everyday.",
-      "Also, developed features responsible for anomaly detection like money leaks, and fraudulent txns in ART.",
-      "Reduced internal data fetching scripts time from greater than 2 hours to under few minutes. With this fetching jobs now work with 10% of initial cost.",
-      "Reduces data ingestion time in spark pipeline by 10-20% by changing few ingestion strategies.",
-      "Developed Reconciliation and Settlement Service module from scratch for ONDC (Open Network for Digital Commerce) initiative.",
-      "Implemented fraud detection mechanism on commission payouts to prevent loss/abuse. This helped identify monthly payouts up to 3.5 million. (Net revenue loss)",
-      "Setup alert and monitoring framework for workers which later helped in detecting org-wide outages before creating impact on payments.",
-      "Developed & Managed commission module for various payment products at scale of 300 TPS for commisions create and at 900 TPS for its pricing.",
-      "Executed successful migrations of critical operations without interruption of services, (user roles migration)",
-      "Managed P2 incident (customer experience breakage) from triage to RCA."
+      "Developed and implemented new features for internal reconciliation tool to process millions of transactions daily including payments, refunds, chargebacks, and settlements.",
+      "Implemented anomaly detection algorithms to identify potential money leaks and fraudulent transactions.",
+      "Optimized data fetching scripts to reduce run time from over 2 hours to under a few minutes, reducing costs by 90%.",
+      "Reduced Spark data ingestion time by 10-20% through ingestion strategy improvements.",
+      "Developed the Reconciliation and Settlement Service module from scratch for the ONDC (Open Network for Digital Commerce) initiative.",
+      "Built a fraud detection system to prevent loss and abuse in monthly commission payouts identifying up to 3.5 million in revenue loss each month. (Net revenue loss)",
```

```suggestion
      "Built a fraud detection system to prevent loss and abuse in monthly commission payouts identifying up to 3.5 million in revenue loss each month.",
```
Maybe this or directly writing net revenue loss?